### PR TITLE
arms and hms

### PR DESCRIPTION
**Pull request by Jdvakil**

### CODE REVIEW

File: agents/v0.6/arms/NPG/FetchReachFixed-v0/21_env=FetchReachFixed-v0,seed=1/.DS_Store
Comment (vikashplus) [inline agents/v0.6/arms/NPG/FetchReachFixed-v0/21_env=FetchReachFixed-v0,seed=1/.DS_Store:1]: remove all DS_Store files

File: build/lib/rlhive/__init__.py
Comment (vikashplus) [inline build/lib/rlhive/__init__.py:1]: What are these files?

File: build/lib/rlhive/envs.py
Comment (vikashplus) [inline build/lib/rlhive/envs.py:1]: What are these files?

File: build/lib/rlhive/rl_envs.py
Comment (vikashplus) [inline build/lib/rlhive/rl_envs.py:1]: What are these files?

File: build/lib/rlhive/version.py
Comment (vikashplus) [inline build/lib/rlhive/version.py:1]: What are these files?

File: rlhive.egg-info/PKG-INFO
Comment (vikashplus) [inline rlhive.egg-info/PKG-INFO:1]: What are these files?

File: rlhive.egg-info/SOURCES.txt
Comment (vikashplus) [inline rlhive.egg-info/SOURCES.txt:1]: What are these files?

File: rlhive.egg-info/dependency_links.txt
Comment (vikashplus) [inline rlhive.egg-info/dependency_links.txt:1]: What are these files?

File: rlhive.egg-info/not-zip-safe
Comment (vikashplus) [inline rlhive.egg-info/not-zip-safe:1]: What are these files?

File: rlhive.egg-info/requires.txt
Comment (vikashplus) [inline rlhive.egg-info/requires.txt:1]: What are these files?

File: rlhive.egg-info/top_level.txt
Comment (vikashplus) [inline rlhive.egg-info/top_level.txt:1]: What are these files?

File: rlhive/version.py
Comment (vikashplus) [inline rlhive/version.py:1]: What are these files?

**Review by vikashplus — COMMENTED** [review]
See comments

**Comment by Jdvakil** [comment]
I think these were generated by codespace, removed them

**Review by vikashplus — APPROVED** [review]

**Pull request merged**